src/harness/testcases/WINNF_FT_S_SSS_testcase.py

```diff
@@ -18,10 +18,67 @@
 
 SAS_CERT = os.path.join('certs', 'sas.cert')
 SAS_KEY = os.path.join('certs', 'sas.key')
+SAS_URL = 'https://fake.sas.url.not.used.org'
+
 
 class SasToSasSecurityTestcase(security_testcase.SecurityTestCase):
   # Tests changing the SAS UUT state must explicitly call the SasReset().
 
+  @winnforum_testcase
+  def test_WINNF_FT_S_SSS_1(self):
+    """New registration with TLS_RSA_WITH_AES_128_GCM_SHA256 cipher.
+
+    Checks that SAS UUT response satisfy cipher security conditions.
+    Checks that a SAS registration with this configuration succeed.
+    """
+    self.doSasTestCipher('AES128-GCM-SHA256',
+                         client_cert=SAS_CERT, client_key=SAS_KEY,
```

Comment:
Should we make the cert and key as configurable parameters instead of global variables?

Reply:
As we don't expect to run these tests with different set of certificates, I think there is no point to use a configurable test case here.

Reply:
Another point is that these tests are checking the algorithm, not the certs, so we can use the default certs the same way we use them for every other test. Let me know your thoughts so that we can go on with this one. Thanks.

Reply:
we do not hold strong position on this comment as long as the test code servers the requirements. If in case we need them be configurable, we can patch fixes.